package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "test": "jest",
     "precommit": "lint-staged",
-    "release": "npm login && release-it"
+    "release": "npm login && release-it",
+    "start": "react-native start"
```

Comment:
Not sure if this is needed, we should run the example from its directory.

Reply:
I added it to be able to clear cache in the main folder because it created conflicts without doing it then.

Reply:
Does cache in the main dir matter? Could you provide an example of such cache conflict?

Reply:
The conflicts appeared when I added the proper resolving of module in the example. I am almost sure it is not easily reproducible and maybe it is not needed indeed.

Reply:
Let's remove it then, we can re-add if neccessary.

Reply:
Changed in 6cadf5c.